docs/framework/network-programming/network-programming-samples.md

```diff
@@ -16,60 +16,49 @@ ms.workload:
   - "dotnet"
 ---
 # Network Programming Samples
-This section contains descriptions and links to downloadable network programming samples that use classes in the <xref:System.Net>, <xref:System.Net.Cache>, <xref:System.Net.Configuration>, <xref:System.Net.Mail>, <xref:System.Net.Mime>, <xref:System.Net.NetworkInformation>, <xref:System.Net.PeerToPeer>, <xref:System.Net.Security>, <xref:System.Net.Sockets>, and related namespaces.  
+This section contains descriptions and links to downloadable network programming samples that use classes in the <xref:System.Net>, <xref:System.Net.Cache>, <xref:System.Net.Configuration>, <xref:System.Net.Mail>, <xref:System.Net.Mime>, <xref:System.Net.NetworkInformation>, <xref:System.Net.PeerToPeer>, <xref:System.Net.Security>, <xref:System.Net.Sockets>, and related namespaces.
+Note: all samples are available in the .NET Framework SDK version 2.0 Samples Download, which was released for the .NET Framework 2.0 and may be dated.
   
 ## In This Section  
- [Download Progress Indicator Technology Sample](http://go.microsoft.com/fwlink/?LinkID=179556)  
+ [Download Progress Indicator Technology Sample](https://msdn.microsoft.com/en-us/library/t8w6294a(v=vs.85).aspx)  
  Shows how to display the progress of a file download.  
   
- [FTP Client Technology Sample](http://go.microsoft.com/fwlink/?LinkID=179557)  
+ [FTP Client Technology Sample](https://msdn.microsoft.com/en-us/library/b7810t5c(v=vs.85).aspx)  
  Shows how to upload and download files to and from an FTP server.  
   
- [HttpListener Technology Sample](http://go.microsoft.com/fwlink/?LinkID=179558)  
+ [HttpListener Technology Sample](https://msdn.microsoft.com/en-us/library/y7cbb2y2(v=vs.85).aspx)  
  Shows how to process HTTP requests from within an application.  
   
- [HttpListener ASPX Host Application Sample](http://go.microsoft.com/fwlink/?LinkID=179560)  
```

Comment:
This sample does actually exist in the .NET Framework SDK version 2.0 Samples, in the Applications\Networking\AspxHost directory. It has a page in the .NET Framework 3.5 doc set, but not in the .NET Framework 3.0 doc set. We could include it: \[HttpListener ASPX Host Application Sample\]\(https://docs.microsoft.com/en-us/previous-versions/visualstudio/visual-studio-2008/dd767375(v%3dvs.90)\)